app/views/templates/Layout.scala.html

```diff
@@ -77,8 +77,8 @@
 
 @content = {
     @contentBlock
-
-    <div class="govuk-!-margin-top-8">
+<input type="hidden" id="spinner-counter" value="@appConfig.spinnerCounter" />
```

Comment:
@shaileshp0110 Instead of layout can we not use it in the specific page where it is required